webpack.config.js

```diff
@@ -60,7 +60,12 @@ module.exports = {
       chunkFilename: "css/[name].css",
     }),
     new webpack.EnvironmentPlugin({
-      API_URL: process.env.npm_package_config_api_url,
+      API_URL:
+        process.env.npm_package_config_api_url ||
+        "https://api.railway-stations.org/", // TODO is this still working as intended?
```

Comment:
@pstorch maybe you could have a look into this, as I never used a locally running api server, I couldn't test it.

Reply:
Looks like I can't override the config of the api_url anymore.

I've added a `echo` script to debug it:
```bash
npm run echo
```
Even when I override it with my `.npmrc` via 

```bash
npm config set railway-stations-frontend:api_url http://192.168.0.229:8080/
```

the `api_url` is still the default as defined in the `package.json`: https://api.railway-stations.org/

@greenhalos any idea? I've also looked into other solutions like dotenv and others but couldn't get it to work.

BTW: the website itself seems to work again, many thanks. Maybe we should already merge that and tackle the local api_url override separately.